web/src/controller/coaching_session_controller.rs

```diff
@@ -0,0 +1,53 @@
+use crate::controller::ApiResponse;
+use crate::extractors::{
+    authenticated_user::AuthenticatedUser, compare_api_version::CompareApiVersion,
+};
+use crate::{AppState, Error};
+use axum::extract::{Query, State};
+use axum::http::StatusCode;
+use axum::response::IntoResponse;
+use axum::Json;
+use entity_api::coaching_session as CoachingSessionApi;
+use service::config::ApiVersion;
+use std::collections::HashMap;
+
+use log::*;
+
+#[utoipa::path(
+    get,
+    path = "/coaching_sessions",
+    params(
+        ApiVersion,
+        ("coaching_relationship_id" = Option<Id>, Query, description = "Filter by coaching_relationship_id"),
+        ("from_date" = Option<NaiveDate>, Query, description = "Filter by from_date"),
```

Comment:
Can we use `DateTime` instead of `NaiveDate`?

Reply:
Bookmarked as a [future discussion](https://github.com/Jim-Hodapp-Coaching/refactor-platform-rs/discussions/51#discussion-6901596).